src/supervisor-api.coffee

```diff
@@ -29,11 +31,22 @@ authenticate = (config) ->
 		.catch (err) ->
 			res.status(503).send("Unexpected error: #{err}")
 
+expressLogger = morgan (tokens, req, res) -> [
+		'SAPI'
```

Comment:
I assume `SAPI` here is meant to stand for Supervisor API?  If so could you make it a bit clearer as SAPI can definitely stand for other things

Reply:
Sure thing.